try-it.md

```diff
@@ -6,30 +6,28 @@ permalink: /try-it.html
 
 <!-- TOC depthFrom:2 insertAnchor:false orderedList:false updateOnSave:true withLinks:true -->
 
-- [1. Instructions](#1-instructions)
+- [1. Environment Setup](#1-environment-setup)
     - [1.1. Prerequisites](#11-prerequisites)
-    - [1.2. Metal3-dev-env setup](#12-metal3-dev-env-setup)
-    - [1.3. Using a custom image](#13-using-a-custom-image)
-- [2. Working with the Environment](#2-working-with-the-environment)
-    - [2.1. Bare Metal Hosts](#21-bare-metal-hosts)
-    - [2.2. Provisioning Cluster and Machines](#22-provisioning-cluster-and-machines)
-    - [2.3. Deprovisioning Cluster and Machines](#23-deprovisioning-cluster-and-machines)
-    - [2.4. Centos target hosts only, image configuration](#24-centos-target-hosts-only-image-configuration)
-    - [2.5. Directly Provisioning Bare Metal Hosts](#25-directly-provisioning-bare-metal-hosts)
+    - [1.2. Setup](#12-setup)
+    - [1.3. Using Custom Image](#13-using-custom-image)
+- [2. Working with Environment](#2-working-with-environment)
+    - [2.1. BareMetalHosts](#21-baremetalhosts)
+    - [2.2. Provision Cluster and Machines](#22-provision-cluster-and-machines)
+    - [2.3. Deprovision Cluster and Machines](#23-deprovision-cluster-and-machines)
+    - [2.4. Image configuration (Centos 7 target hosts only)](#24-image-configuration-centos-7-target-hosts-only)
+    - [2.5. Directly Provision BareMetalHost](#25-directly-provision-baremetalhost)
     - [2.6. Running Custom Baremetal-Operator](#26-running-custom-baremetal-operator)
     - [2.7. Running Custom Cluster API Provider Metal3](#27-running-custom-cluster-api-provider-metal3)
     - [2.8. Accessing Ironic API](#28-accessing-ironic-api)
 
 <!-- /TOC -->
 <hr>
-## 1. Instructions
+## 1. Environment-Setup
```

Comment:
This looks like the only title with a `-` in it. Can we remove that for consistency?

Reply:
Sure, fixed.